lib/assert/element.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+var util = require('util');
+
+var assert = require('assertive');
+var _ = require('lodash');
+
+function isTextOrRegexp(textOrRegExp) {
+  return _.isString(textOrRegExp) || _.isRegExp(textOrRegExp);
+}
+
+exports._getElementWithProperty = function _getElementWithProperty(selector, property) {
+  var element = this._getElement(selector);
+  return [ element, element.get(property) ];
+};
```

Comment:
Good point. Will switch to inlining. One of the things I should have questioned instead of blindly porting it 1:1.